CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ requesting a new account and will be displayed in the User Accounts module (PR #
 #### Bug Fixes
 - *Add item here*
 ### Modules
+#### Help Editor
+- parentID was removed. (PR #7025)
```

Comment:
I don't think this changelog update means anything to users.

Reply:
What do you suggest?

Reply:
how about a few more words to make it meaningful to readers, like : 
> Cleaned up deprecated column `parentID`  #7025

Reply:
I still don't think that means anything to readers. What's the effect of removing parentID? If it's not noticeable in any way, it's not worth mentioning in the changelog.

Reply:
It's only affecting the rendering of the data table, this change remove the column Parent Topic
![Screenshot from 2020-10-05 15-22-21](https://user-images.githubusercontent.com/32041423/95122651-d3472300-071e-11eb-89da-75b5af90e5ca.png)

Reply:
@driusan I edited the CHANGELOG to add @christinerogers suggestion, let me know if you prefer to remove the mention.